test/e2e/scenarios/ACL/acl_update_with_tags.yaml

```diff
@@ -54,13 +54,20 @@ steps:
           - key: "test_key_1"
             value: "test_value_1"
           - key: "test_key_2"
-          - key:
+            value: "test_value_2"
     wait:
       status:
         conditions:
           ACK.ResourceSynced:
             status: "True"
             timeout: 100
+    expect:
+      spec:
+        tags:
+          - key: "test_key_1"
+            value: "test_value_1"
+          - key: "test_key_2"
+            value: "test_value_2"
```

Comment:
Looks like this new solution already found a bug!

Reply:
Actually this is not a bug. I checked MemoryDB APIs, and MemoryDB doesn't support a null key for tags (ideally each tag should have both key and value). For non-null key with null value case, MemoryDB automatically fills a null value for the key. I've just added value for this case so that it won't confuse others.